src/kps/gui/windows/form/dialogs/RouteDialog.java

```diff
@@ -51,8 +76,6 @@ public RouteDialog(@Nullable Frame owner, @NotNull Simulation simulation, boolea
         setVisible(true);
     }
 
-
-
     @Override protected void initializeForm(FormBuilder builder) {
```

Comment:
Ive commented on the parent class. and I didnt want to touch protected and private methods otherwise ill be here all day